server/service/src/sync/api_v6/core.rs

```diff
@@ -0,0 +1,107 @@
+use crate::sync::{api::ParsingResponseError, SyncApiSettings};
+use reqwest::{Client, Response};
+use thiserror::Error;
+use url::{ParseError, Url};
+
+use super::*;
+
+// For now port_offset of 2
+static PORT_OFFST: u16 = 2;
```

Comment:
Still interested in another solution rather than port offset. Doesn't look too hard to plumb another url through IMHO.
Even if we default to the port offset, but allow it to be set in a configuration that would be a good thing.
I think having a separate port as an option will make dev's life much easier. We don't all want to have to setup Caddy or something...

If rust could be the proxy for old mSupply that would be a good solution. I'm keen for a follow up issue, and happy to experiment with options if you'd like.

Reply:
If I understood correctly, we have the use-case that we need to run the remote server and the new central server on the same machine, and then we need two ports, don't we?

Think as long as we can configure the url/port we can decide later how we deploy it and what is best for a dev setup.

Reply:
I think the main reason i went with port offset for now, was because of existing sites. How do we point them to the omSupply central server instance unless we caddy or port offset ? 

> I'm keen for a follow up issue, and happy to experiment with options if you'd like.

Sure although I am wondering for this feature to go through, what is the minimum that is needed now, and what is the strategy for existing sites for later ?

Use case, existing deployment, and we upgrade them to omSupply version that requires omSupply central (not the pack variant version but say the assets version). We don't do the port offset thing, or the caddy thing, how do we tell all remotes sites that server is now somewhere else ? 

(I think we would need classic mSupply central to expose API to query for omSupply central server location, and this would happen as the first step of sync)